### PR TITLE
Create Question_ep3_pitfalls.md

### PR DESCRIPTION
**Pull request by TerpstraS**

### CODE REVIEW

File: Assignments/Podcast_reflections/Question_ep3_pitfalls.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	+ Doing a randomized controlled experiment between research with and without open science practices.
+8. Open science and recognition & rewards are similar concepts.
+	+ True
+	+ False
+9. What outcomes could one evaluate to assess the benefits of open science?
+	+ Reproducibility
+	+ Transparency
+	+ Both of the above
+	+ None of the above
+10. Does engaging in open science practices require creating new jobs in academia?
```
Comment (SanliFaez) [inline Assignments/Podcast_reflections/Question_ep3_pitfalls.md:46]: I am not sure I understand this formulation of the question. Engaging by who? I don't exactly remember the context of the question, but in this form it reads like a rhetorical question

**Review by SanliFaez — CHANGES_REQUESTED** [review]
please reconsider the formulation of Q10

File: Assignments/Podcast_reflections/Question_ep3_pitfalls.md (hunk 1 above)
Comment (TerpstraS) [reply Assignments/Podcast_reflections/Question_ep3_pitfalls.md:46]: I am not able to listen to the podcast now, but I think the point was that proper implementation of open science requires new jobs in academia. We could rephrase it to "Do open science practices require creating new jobs in academia?", or we can take the question out?

Comment (SanliFaez) [reply Assignments/Podcast_reflections/Question_ep3_pitfalls.md:46]: I have tried different formulations myself but it always becomes a bit too simplistic. There are certain things that can be put in every person's work routine and other things that will require specialization. I find this topic always mixed with a bit of non-sequitur

**Review by SanliFaez — APPROVED** [review]
